packages/stripe/src/Adaptor.js

```diff
@@ -0,0 +1,100 @@
+import { expandReferences } from '@openfn/language-common/util';
+import * as util from './Utils';
+
+/**
+ * State object
+ * @typedef {Object} HttpState
+ * @property data - the parsed response body
+ * @property response - the response from the HTTP server, including headers, statusCode, body, etc
+ * @property references - an array of all previous data objects used in the Job
+ **/
+
+/**
+ * List items from Stripe
+ * @example <caption>List all payment intents</caption>
+ * list('payment_intents');
+ * @example <caption>List customers with options</caption>
+ * list('customers', {limit:3});
+ * @function
+ * @public
+ * @param {string} resource - The API resource to list items from.
+ * @param {object} options -  Optional options object. See {@link https://docs.stripe.com/api/payment_intents/list|Stripe API documentation} for more details.
+ * @returns {Operation}
+ * @state {HttpState}
```

Comment:
Need to add `@state data - an array of items` to make it clear what state.data looks like